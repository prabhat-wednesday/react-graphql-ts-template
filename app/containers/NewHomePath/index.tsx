import React from 'react';

const NewHomePath = () => {
  return <h1>New Home Path</h1>;
};

export default NewHomePath;
