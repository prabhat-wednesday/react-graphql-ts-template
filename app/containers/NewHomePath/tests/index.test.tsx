import React from 'react';
import { render } from '@testing-library/react';
import NewHomePath from '..';

describe('<NewHomePath /> test', () => {
  it('should render and match the snapshot', () => {
    const { baseElement } = render(<NewHomePath />);
    expect(baseElement).toMatchSnapshot();
  });
});
