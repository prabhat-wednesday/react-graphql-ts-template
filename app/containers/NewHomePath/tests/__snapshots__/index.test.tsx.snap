// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewHomePath /> test should render and match the snapshot 1`] = `
<body>
  <div>
    <h1>
      New Home Path
    </h1>
  </div>
</body>
`;
